torch/_torch_docs.py

```diff
@@ -2358,6 +2363,67 @@ def parse_kwargs(desc):
     tensor([ 1.2252,  0.5002,  0.6248,  2.0139])
 """)
 
+add_docstr(torch.matrix_diag,
+           r"""
+matrix_diag(input, offset=0, dim1=-2, dim2=-1) -> Tensor
+
+Creates a diagonal matrix from :attr:`input` by using the last
```

Comment:
This first sentence is a bit unclear. It usually creates more than 1 matrix, or even the created tensors are not matrices. I also have some questions about the name and API raised above.